.rspec

```diff
@@ -0,0 +1,2 @@
+--format documentation
```

Comment:
I'd prefer not this one because it's so damn noisy. I only want to see the failures usually.